<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WordInfoDialog</class>
 <widget class="QDialog" name="WordInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>주제</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="themeList"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>정보</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="searchedWordList"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="inputTheme"/>
         </item>
         <item>
          <widget class="QPushButton" name="search2">
           <property name="text">
            <string>검색</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
